modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -289,6 +293,8 @@ export default class AgentManager {
 
                 this.connectionSbj.next(conn);
 
+                this.saveToStorage();
```

Comment:
Why we save on each update? We save on change

Reply:
Now implemented saving only on cluster switch. When cluster is single it will never be saved.